climakitae/core/data_interface.py

```diff
@@ -946,122 +1035,148 @@ def _update_unit_options(self):
-            self.scenario_historical = _scenario_historical
+
+            # check if new selection has the historical scenario options and if not select the first new option
+            if _check_inputs(_scenario_historical, scenario_historical_options):
+                self.scenario_historical = _scenario_historical
+            else:
+                self.scenario_historical = [scenario_historical_options[0]]
+
         else:
-            self.scenario_historical = [scenario_historical_options[0]]
+            pass
 
     @param.depends(
+        "approach",
         "scenario_ssp",
         "scenario_historical",
         "downscaling_method",
         "time_slice",
         watch=True,
     )
     def _update_data_warning(self):
-        """Update warning raised to user based on their data selections."""
+        """Update warning raised to user based on their data selections.
+        No warming shown if approach is set to warming level.
+        """
         data_warning = ""
         bad_time_slice_warning = """You've selected a time slice that is outside the temporal range 
         of the selected data."""
 
-        # Set time range of historical data
-        if self.downscaling_method == "Statistical":
-            historical_climate_range = self.historical_climate_range_loca
-        elif self.downscaling_method == "Dynamical+Statistical":
-            historical_climate_range = self.historical_climate_range_wrf_and_loca
-        else:
-            historical_climate_range = self.historical_climate_range_wrf
-
-        # Warning based on data scenario selections
-        if (  # Warn user that they cannot have SSP data and ERA5-WRF data
-            True in ["SSP" in one for one in self.scenario_ssp]
-        ) and ("Historical Reconstruction" in self.scenario_historical):
-            data_warning = """Historical Reconstruction data is not available with SSP data.
-            Try using the Historical Climate data instead."""
+        if self.approach == "Warming Level":
+            data_warning = ""
 
-        elif (  # Warn user if no data is selected
-            not True in ["SSP" in one for one in self.scenario_ssp]
-        ) and (not True in ["Historical" in one for one in self.scenario_historical]):
-            data_warning = "Please select as least one dataset."
+        else:
+            # Set time range of historical data
+            if self.downscaling_method == "Statistical":
+                historical_climate_range = self.historical_climate_range_loca
+            elif self.downscaling_method == "Dynamical+Statistical":
+                historical_climate_range = self.historical_climate_range_wrf_and_loca
+            else:
+                historical_climate_range = self.historical_climate_range_wrf
+
+            # Warning based on data scenario selections
+            if (  # Warn user that they cannot have SSP data and ERA5-WRF data
+                True in ["SSP" in one for one in self.scenario_ssp]
```

Comment:
I know you didn't write this in this PR, but a potential small change to include for this PR or another is:
`any("SSP" in one for one in self.scenario_ssp)`